aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1379,6 +1427,34 @@ module aptos_token::token {
         assert!(property_map::length(&token.token_properties) == 2, property_map::length(&token.token_properties));
     }
 
+    #[test(creator = @0xAF, owner = @0xBB)]
+    fun test_mutate_token_uri(creator: &signer, owner: &signer) acquires Collections, TokenStore {
+        account::create_account_for_test(signer::address_of(creator));
+        account::create_account_for_test(signer::address_of(owner));
+
+        // token owner mutate the token property
+        let token_id = create_collection_and_token(creator, 1, 1, 1);
+        assert!(token_id.property_version == 0, 1);
+
+        update_token_uri(
+            creator,
+            signer::address_of(creator),
+            token_id,
+            string::utf8(b"https://revealed.io")
+        );
+
+        // let creator_props = &borrow_global<TokenStore>(signer::address_of(creator)).tokens;
+        // let token = table::borrow(creator_props, new_id_1);
+
+        let uri = get_token_uri(signer::address_of(creator), token_id.token_data_id);
+        assert!(uri == string::utf8(b"https://revealed.io"), 999);
+
+        // // transfer token with property_version > 0 also transfer the token properties
```

Comment:
Is this code still used? If not, please delete.

Reply:
this is't resolved.